tools/docker/tests/openwrt/test_status.sh

```diff
@@ -24,15 +42,29 @@ for _ in $(seq 1 "$TIMEOUT") ; do
 done
 exit 1
 EOF
-
-# Save exit status and capture the logs
+# Save exit status
 TEST_STATUS=$?
+
+# Collect post-run diagnostics
+echo "Collecting post-test-run diagnostic data..."
+ssh "$TARGET" <<"EOF" >> "${TARGET}_diags.log"
+date
+echo "Output of bml_conn_map:"
+/opt/prplmesh/bin/beerocks_cli -c bml_conn_map 2>&1
+echo "Output of bml_nw_map_query:"
+/opt/prplmesh/bin/beerocks_cli -c bml_nw_map_query 2>&1
```

Comment:
What is `bml_nw_map_query`?

Reply:
It's at least being used by the analyzer, and from what I recall its output can be different than `bml_conn_map`.
I don't know to what extend it's relevant to have it in the archive though.

@dob71 do you have a use case where `bml_nw_map_query` is useful?